package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-modal",
-  "version": "3.14.4",
+  "version": "3.15.1",
```

Comment:
Looks like this was slightly outdated, this was automatically changed when installing dependencies locally. I've left the change in here, but am happy to revert too

Reply:
This is done automatically when a release is made. 3.15.1 is kinda broken...you can revert this.

Reply:
Done in [f88b5a0](https://github.com/reactjs/react-modal/pull/953/commits/f88b5a0ec75c0dd4b485a5ff02b5b752429aae59)